### DISCUSSION TITLE
How to view files/folders in a git folder

### DISCUSSION THREAD

**Discussion by lianhuilui** [LazyVim/LazyVim#1431] [Q&A]
If I open a directory with .git, e.g. a project, I cannot view any files or folders.
If I open the parent directory and then go into the said directory, it works fine.
(default config)

**Comment by dpetka2001** [comment]
I cannot reproduce this. What are you using to view the files?

**Reply by lianhuilui** [reply]
The default, neo-tree. The folder icon is hollow, as well, which must indicate something meaningful.
![image](https://github.com/LazyVim/LazyVim/assets/2843170/3f0ff6b4-92ad-4c9a-8842-114a3ac2c037)

When I run nvim in the parent folder and press . to cwd, it works fine.
![image](https://github.com/LazyVim/LazyVim/assets/2843170/784ee294-e3c2-4675-b8a0-ccf8ddcc4499)

**Reply by dpetka2001** [reply]
Maybe it has to do something with your `.gitignore`? I'm still not able to reproduce it. When I'm inside a `.git` repo and I open Neovim, if I open `neo-tree` it shows the files correctly.

**Reply by lianhuilui** [reply]
I do not have a .gitignore. Must be some kind of conflict with the other plugins or some packages there were installed previously. It's alright. I'm starting a new config from kickstart anyway. Thanks for your quick replies!
